docs/fsharp/style-guide/formatting.md

```diff
@@ -870,40 +953,45 @@ let rainbow =
       Lackeys = ["Zippy"; "George"; "Bungle"] }
 ```
 
-Long record field expressions should use a new line and have one indent from the opening `{`:
+#### Multiline bracket formatting styles
 
-```fsharp
-{ A = a
-  B =
-    someFunctionCall
-        arg1
-        arg2
-        // ...
-        argX
-  C = c }
-```
+For records that span multiple lines, there are three commonly used formatting styles: `Cramped`, `Aligned`, and `Stroustrup`. The `Cramped` style has been the default style for F# code, as it tends to favor styles that allow the compiler to easily parse code. Both `Aligned` and `Stroustrup` styles allow for easier reordering of members, leading to code that may be easier to refactor, with the drawback that certain situations may require slightly more verbose code.
 
-Placing the `{` and `}` on new lines with contents indented is possible, however code formatters may reformat this by default:
+* `Cramped` Record style:
+The historical standard, and default F# record format. Opening brackets go on the same line as the first member, closing bracket on the same line as the last member.
 
-```fsharp
-// ✔️ OK
-let rainbow =
-    { Boss1 = "Jeffrey"
-      Boss2 = "Jeffrey"
-      Boss3 = "Jeffrey"
-      Lackeys = ["Zippy"; "George"; "Bungle"] }
+    ```fsharp
+    let rainbow = 
```

Comment:
I don't think these code blocks need to be indented.

Reply:
Even though they're part of the list items?

Reply:
Oh, that's why. Well maybe, I don't know, it will be up to the people reviewing this.
It could be either way in this case.